src/utils/composeStores.js

```diff
@@ -1,8 +1,17 @@
 import mapValues from '../utils/mapValues';
 import pick from '../utils/pick';
+import invariant from 'invariant';
 
 export default function composeStores(stores) {
   const finalStores = pick(stores, (val) => typeof val === 'function');
+
+  Object.keys(finalStores).forEach(key => {
+    invariant(
+        typeof finalStores[key]({}, {}) !== 'undefined',
```

Comment:
Okay, I think I'm confident now I only want to do this in development. Let's add `process.env.NODE_ENV !== 'production'` check and use `console.warn` instead.